articles/quickstart/native/react-native/00-login.md

```diff
@@ -181,16 +136,64 @@ and then register a URL type entry using the value of `CFBundleIdentifier` as th
         <string>auth0</string>
         <key>CFBundleURLSchemes</key>
         <array>
-            <string>org.reactjs.native.example.$(PRODUCT_NAME:rfc1034identifier)</string>
+            <string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
         </array>
     </dict>
 </array>
 ```
 
 ::: note
-The value org.reactjs.native.example.$(PRODUCT_NAME:rfc1034identifier) is the default for apps created with React Native CLI, you may have a different value.
+If your application was generated using the React Native CLI, the default value of `$(PRODUCT_BUNDLE_IDENTIFIER)` dynamically matches `org.reactjs.native.example.$(PRODUCT_NAME:rfc1034identifier)`. For the sample app, this value matches `com.auth0samples`.
 :::
 
+Take note of this value as you'll be using it to define the callback URLs below. If desired, you can change it using XCode in the following way:
+
+- Open the `ios/TestApp.xcodeproj` file replacing 'TestApp' with the name of your app or run `xed ios` on a Terminal from the app root.
```

Comment:
Consider: "Open the `ios/TestApp.xcodeproj` file and replace 'TestApp' with the name of your app or run `xed ios` on a Terminal from the app root."

Reply:
I think your suggestion changes a bit the reading. My intention is to say open `ios/{your app name}.xcodeproj`. That's why I'm saying "open ios/myapp replacing 'myapp' with your app name". Now that you have the context, would you rewrite it in a different way @jimmyjames ?